MediaBrowser.Model/Cryptography/PasswordHash.cs

```diff
@@ -16,70 +16,78 @@ public class PasswordHash
         public byte[] SaltBytes;
         public string Hash;
         public byte[] HashBytes;
-        public PasswordHash(string StorageString)
+        public PasswordHash(string storageString)
         {
-            string[] a = StorageString.Split('$');
-            Id = a[1];
-            if (a[2].Contains("="))
+            string[] SplitStorageString = storageString.Split('$');

```

Comment:
Maybe rename `SplitStorageString` to just `splitted`?

Reply:
don't really care what its called so renamed splitted. done.